doc/security_policies.md

```diff
@@ -20,68 +20,85 @@ We can classify rules into multiple cases:
 
 ## YaST and Security Policies
 
-The YaST installer supports security policies which can be enabled in the Installation Summary. The
-installer checks the rules of the enabled policies and reports the issues to fix before proceeding
-with the installation. For some issues, YaST offers a link to easily remediate it. For others, the
-link will take you to the proper installation client to fix the issue (e.g., the Guided
-Partitoning). At this moment, only the DISA STIG policy is supported, but the YaST infrastructure
-is ready to offer more policies if requested.
-
-The boot parameter `YAST_SECURITY_POLICIES` can be used for enabling policies from the beginning of
-the installation without waiting until reaching the Installation Summary. If the policies are
-enabled (e.g., `YAST_SECURITY_POLICIES=disa_stig`), then some YaST clients could show policies
-issues too. For example, the Guided Partitioning client and the Expert Partitioner show the policies
-issues related to storage configuration.
+The YaST installer supports security policies which can be enabled in the Installation Summary.
```

Comment:
Not only in the installation summary: you can pass a boot argument or use AutoYaST. What about "YaST installer supports checking security policies..." or something similar.